ipv8/community.py

```diff
@@ -254,7 +254,7 @@ def create_introduction_response(self, lan_socket_address, socket_address, ident
         if introduced:
             packet = self.create_puncture_request(lan_socket_address, socket_address, identifier, prefix=prefix,
                                                   new_style=new_style)
-            self.endpoint.send(introduction_wan if introduction_lan == ("0.0.0.0", 0) else introduction_lan, packet)
+            self.endpoint.send(introduction.address, packet)
```

Comment:
Would this fix https://github.com/Tribler/anydex-core/issues/64? 🤔

Reply:
Yes, it should. That's how I discovered the bug.

Reply:
Great, thanks! I was unable to find out where it originates from so I just disabled that test 😂